src/main.js

```diff
@@ -53,40 +60,75 @@ function randomizePoster() {
   secondDescriptor.innerText = currentCover.tagline2;
 }
 
-function createCustomCover() {
-  currentCover = new Cover(
-    userCover.value,
-    userTitle.value,
-    userDescriptorOne.value,
-    userDescriptorTwo.value
-  );
-  // covers.unshift(userCover.value);
-  // titles.unshift(userTitle.value);
-  // descriptors.unshift(userDescriptorOne.value);
-  // descriptors.unshift(userDescriptorTwo.value);
-}
-
-function displayCustomCover() {
-  cover.src = currentCover.cover;
-  title.innerText = currentCover.title;
-  firstDescriptor.innerText = currentCover.tagline1;
-  secondDescriptor.innerText = currentCover.tagline2;
+function noDuplicates() {
+  //IF the currentCover = item in savedCovers, then do not save
+  //ELSE
+
+  var duplicates;
+   for (var i = 0; i < savedCovers.length; i++) {
+//      if(savedCovers.indexOf(currentCover) === -1) {
+//        return false;
+//      } else {
+//        savedCovers.unshift(currentCover);
+//      }
+//   }
+// }
+    if(savedCovers[i] === currentCover) {
+      duplicates = true;
+    } else {
+      duplicates = false;
+    }
+    return duplicates;
+  }
+}
+
+  // if(savedCovers.includes(currentCover)) {
+  //   return false;
+  // } else {
+  //   savedCovers.unshift(currentCover);
+  // }
+// }
+
+function saveCover(){
+  noDuplicates()
```

Comment:
I could be wrong but it looks like the first `noDuplicates` call isn't doing anything for you. It seems like you're calling it correctly in the if statement, which is good!

Reply:
Oh, good call.  Thanks for catching that!

Reply:
I think you're definitely right. We'll delete that line out & see where it takes us!